dataland-frontend/src/components/forms/parts/UploadReports.vue

```diff
@@ -152,6 +110,7 @@ export default defineComponent({
     },
   },
   methods: {
+    removeFileTypeExtension,
```

Comment:
Doesn't this need a description?

Reply:
No, it is an imported function and has a jsdoc description located at its declaration.

Reply:
ok